lib/src/script.dart

```diff
@@ -238,6 +253,10 @@ class Script {
   ///
   /// The [name] is a human-readable name for the script, used for debugging and
   /// error messages. It defaults to `"capture"`.
+  ///
+  /// The script can only be interrupted by closing the [Script.stdin] stream.
+  /// Any [signal] is ignored and returns whether or not the script is sill
+  /// running.
```

Comment:
I think we should give `Script.capture()` callers an optional argument to explicitly handle signals. The expected semantics are substantially different than closing stdin (which may already be a meaningful indicator of just "end of data").

Reply:
For capturing instead of terminating the stdout (which would not terminate the callback itself) I added an `onSignal` which users can then use if they want cancellation logic in their script